### ISSUE TITLE
How to use brackets?

### ISSUE THREAD

**Issue by moshpirit** [federico-terzi/espanso#2118]
### Description

I've been trying to use [] and {} but I can't for some reason:

```
  - trigger: ";awk"
    replace: " awk -F= '{print $2}'"
  - trigger: ";[code]"
    replace: "[code]$|$[/code]"
```
For example, for `;awk` I get:  `awk -F= '7print $2/'` and `;[code]` into `8code98/code9` I think that it might be related to my layout (Spanish), but I'm not sure.

### Logs

Posted here: http://0x0.st/XGLW.txt

### Your environment

- Public source code: I don't know what to answer here
- Environment name and version (e.g., Rust 1.59.0): I don't know what to answer here
- Operating system and version (e.g., Ubuntu 20.04.2 LTS): Arch linux
- Espanso version (e.g., 2.1.4-beta): espanso 2.2.1

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by smeech** [comment]
Yes, that may relate to your layout. Here I get: ` awk -F= '{print $2}'` and `[code][/code]` (with the cursor in the middle).
There are many Issues with non-English keyboards.
Linux Xfce, UK keyboard.

**Comment by moshpirit** [comment]
Thanks for replying, @smeech! Any idea on how to solve it? I thought about piping it through an echo command, but haven't tried it yet.

**Comment by smeech** [comment]
No - I have a long list of possibly related issues [here](https://github.com/espanso/espanso/issues/2088#issuecomment-2393536376).
